samples/extensions/dynamic_primitive_clipping/README.adoc

```diff
@@ -0,0 +1,103 @@
+////
+- Copyright (c) 2024, Mobica Limited
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+////
+
+== Dynamic primitive clipping
```

Comment:
Header level should be one level higher, so instead of '==' use '='. Otherwise our docs site build process will output a warning due to a mismatch in header levels.

Reply:
Done.